src/features/content-explorer/content-explorer/__tests__/ContentExplorer.test.js

```diff
@@ -571,4 +599,72 @@ describe('features/content-explorer/content-explorer/ContentExplorer', () => {
             });
         });
     });
+
+    describe('handleSelectAllClick()', () => {
+        const items = [
+            { id: 'item1', name: 'name1' },
+            { id: 'item2', name: 'name2' },
+        ];
+        const selectedItems = { item1: { id: 'item1', name: 'name1' }, item2: { id: 'item2', name: 'name2' } };
+
+        test('should add items to selectedItems when selectAll is called', () => {
+            const wrapper = renderComponent({ items });
+            const result = wrapper.instance().selectAll();
+
+            expect(result).toStrictEqual(selectedItems);
+        });
+
+        test('should remove items from selectedItems when unSelectAll is called', () => {
+            const wrapper = renderComponent({ items });
+            wrapper.setState({ selectedItems });
+            const result = wrapper.instance().unSelectAll();
+
+            expect(result).toStrictEqual({});
+        });
+
+        test('should call selectAll when handleSelectAllClick and checkbox is not selected', () => {
+            const wrapper = renderComponent({ items });
+            wrapper.instance().setState({ isSelectAllChecked: false });
```

Comment:
Since `wrapper.instance()` is used multiple times, can you save it as a variable?